src/generic/grid.cpp

```diff
@@ -9117,26 +9161,28 @@ void wxGrid::SetCornerLabelTextOrientation( int textOrientation )
 
 void wxGrid::SetRowLabelValue( int row, const wxString& s )
 {
-    if ( m_table )
+    if ( m_table && m_rowLabelWidth > 0 )
```

Comment:
Sorry, this is wrong, setting the value should work, i.e. should update the table, even if the labels are currently hidden.

Reply:
Oops, sorry!